app/models/partner.rb

```diff
@@ -228,6 +240,14 @@ def flipper_id
     "Partner;#{id}"
   end
 
+  def partials_to_show
+    displayable_partials || ALL_PARTIALS
+  end
+
+  def displayable_partials
+    PartnerForm.find_by(diaper_bank_id: diaper_bank_id)&.sections
```

Comment:
```suggestion
    PartnerForm.find_by(diaper_bank_id: diaper_bank_id)&.sections
```
Can we can add the `has_one` association to get this instead of putting a query here?
```
  has_one :partner_form, primary_key: :diaper_bank_id, foreign_key: :diaper_bank_id
```
^ This should work and allow you to use `partner_form` in place of the `Partner.find_by(...)`